notebook/gateway/handlers.py

```diff
@@ -60,14 +72,24 @@ def authenticate(self):
             self.log.warning("No session ID specified")
 
     def initialize(self):
-        self.log.info("Initializing websocket connection %s", self.request.path)
         self.session = Session(config=self.config)
-        self.gateway = GatewayWebSocketClient(gateway_url=GatewayClient.instance().url)
 
     @gen.coroutine
     def get(self, kernel_id, *args, **kwargs):
+        """
+        Get method to get the kernel instance from the kernel id.
+        """
+        self.log.info(f'kernel_id in webhook handler get={kernel_id} get')
```

Comment:
There is a two "get" word. It is expected?

Reply:
No no, but this will be changed with the ticket mentioned in the function's doc string.